superset/viz.py

```diff
@@ -373,6 +375,15 @@ def query_obj(self) -> QueryObjectDict:  # pylint: disable=too-many-locals
         self.from_dttm = from_dttm
         self.to_dttm = to_dttm
 
+        # validate sql filters
+        for param in ("where", "having"):
```

Comment:
optional: but this could be DRY(er)

Reply:
I know.. I considered splitting this out into a util, but then decided not to as I'm still hopeful we can remove this file some day, in which case the util would no longer be needed. But happy to follow up and DRY it up if people feel strongly about this.